File: human-readable-time_5kyu/solution.py
def make_readable(s):
    hour = str(s//3600).rjust(2, "0")
    min = str((s//60)%60).rjust(2, "0")
    sec = str(s%60).rjust(2, "0")
    return hour + ":" + min + ":" + sec

File: human-readable-time_5kyu/solution.cs
public static class TimeFormat
{
    public static string GetReadableTime(int seconds)
    {
        var hour = seconds/3600;
        var min = (seconds/60)%60;
        var sec = seconds % 60;
        return hour.ToString().PadLeft(2, '0') +":"+ min.ToString().PadLeft(2, '0') +":"+ sec.ToString().PadLeft(2, '0');
    }
}